build/localnet/manifests/service-accounts.yaml

```diff
@@ -3,3 +3,9 @@ kind: ServiceAccount
 metadata:
   name: debug-client-account
   namespace: default
+---
+apiVersion: v1
```

Comment:
NIT: You use `v1` in some places and the full `rbac.authorization.k8s.io/v1` in others.

I realize they both point to the stable version but figured we could be consistent.

Reply:
It doesn't work. One of the quirks of K8S I guess, cc: @okdas 

I was getting `kubernetes apply: error mapping /Role: no matches for kind "Role" in version "v1"`

I stole the examples from here https://kubernetes.io/docs/reference/access-authn-authz/rbac/